syntax = "proto3";
package iroha.ordering.proto;

import "block.proto";
import "proposal.proto";
import "google/protobuf/empty.proto";

service OrderingGateTransportGrpc {
  rpc onProposal (protocol.Proposal) returns (google.protobuf.Empty);
}

service OrderingServiceTransportGrpc {
  rpc onTransaction (iroha.protocol.Transaction) returns (google.protobuf.Empty);
}
